AEPCore/Mocks/PublicTestUtils/InstrumentedExtension.swift

```diff
@@ -16,6 +16,7 @@ import AEPCore
 import AEPServices
 
 /// Instrumented extension that registers a wildcard listener for intercepting events in current session. Use it along with `TestBase`
+@available(*, deprecated, message: "Will be replaced by a new wildcard listener class soon. Please avoid using InstrumentedExtension directly in your tests.")
```

Comment:
A new class for capturing events using MobileCore.registerEventListener will be added in the next version. Please avoid using InstrumentedExtension directly in your tests.